utils/cppcheck.sh

```diff
@@ -26,26 +26,20 @@ cppcheck "$@" \
   -I build/lib \
   -I enterprise \
   -I lib \
```

Comment:
It will be nice to add `-I 3rdParty/iresearch/core` and probably for vpack too

Reply:
No objections, but I think we should do this separately.
This PR is just meant to be a cleanup PR, which removes obsolete stuff. That should not have an effect on the cppcheck runtime or its results.

However, adding further include directories or adding more files to be test may have an effect on the cppcheck runtime or on the reported results. That's why I would like to do this separately.